include/unk_0203A6DC.h

```diff
@@ -12,28 +12,28 @@
 int FieldPlayerState_SaveSize(void);
 int FieldOWState_SaveSize(void);
 void FieldOWState_Init(UnkStruct_0203A79C * param0);
```

Comment:
I didn't see this till just now but I called 'FieldOWState' 'FieldStatus' instead, will go with whichever name is prefered

Reply:
I would prefer `FieldOverworldState` or something to that effect; I assume this refers to the overall state of the overworld and all of its objects.